### PR TITLE
Add bicycle charging station amenity.

### LINKED ISSUE #290: Render bicycle charging station
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station

amenity=charging_station
bicycle=*

lot of them in Netherland/Germany

### PR DESCRIPTION
**Pull request by Florimondable**
Remove useless test for motorcycle_parking (data is already filter by the where before).
Fix #290

### CODE REVIEW

File: project.mml [hunk 1]
```diff
@@ -1529,6 +1529,7 @@ Layer:
           OR tags->'compressed_air'='yes'
           OR tags->'car_wash'='yes'
           OR (amenity='motorcycle_parking' AND (bicycle='yes' OR bicycle='designated'))
+          OR (amenity='charging_station' AND bicycle='yes')
```
Comment (Phyks) [inline project.mml]: `bicycle='designated'` as well (future-proof, not very important at the moment apparently)?

File: project.mml [hunk 2]
```diff
@@ -1815,6 +1817,7 @@ Layer:
             OR tags->'compressed_air'='yes'
             OR tags->'car_wash'='yes'
             OR (amenity='motorcycle_parking' AND (bicycle='yes' OR bicycle='designated'))
+            OR (amenity='charging_station' AND bicycle='yes')
```
Comment (Phyks) [inline project.mml]: Same comment

File: project.mml [hunk 3]
```diff
@@ -1968,6 +1971,7 @@ Layer:
...
```
Comment (Phyks) [inline project.mml]: Same comment

File: project.mml [hunk 4]
```diff
@@ -1666,6 +1667,7 @@ Layer:
           OR tags->'compressed_air'='yes'
           OR tags->'car_wash'='yes'
           OR (amenity='motorcycle_parking' AND (bicycle='yes' OR bicycle='designated'))
+          OR (amenity='charging_station' AND bicycle='yes')
```
Comment (Phyks) [inline project.mml]: Same

**Comment by Phyks** [comment]
I'm merging and handling the changes myself.

File: project.mml (hunk 1 above)
Comment (Florimondable) [reply project.mml]: oui, j'y ai pensé, mais il y en a quatre dans le monde, donc bof

**Pull request merged**